cmake/FindONNXRuntime.cmake

```diff
@@ -0,0 +1,40 @@
+# FindONNXRuntime
+# ===============
+#
+# Find an ONNX Runtime installation.
+# ONNX Runtime is a cross-platform inference and training machine-learning
+# accelerator.
+#
+# Input variables
+# ---------------
+# 
+#   ONNXRuntime_ROOT            Set root installation.
+#
+# Output variable
+# ---------------
+# 
+#   ONNXRuntime_FOUND           True if headers and requested libraries were found
+#   ONNXRuntime_LIBRARIES       Component libraries to be linked.
+#   ONNXRuntime_INCLUDE_DIRS    Include directories.
+
+find_library(ORT_LIB onnxruntime
+    CMAKE_FIND_ROOT_PATH_BOTH)
+find_path(ORT_INCLUDE onnxruntime/core/session/onnxruntime_cxx_api.h
```

Comment:
This path looks like a little issue because it does not match the directory structure of the official release package where I find everything is in one directory
![grafik](https://user-images.githubusercontent.com/15868645/127781607-4e436f5a-f275-43b0-9aba-14945af8877f.png)
Could you have both paths searched?